src/Core/Content/Cms/SalesChannel/CmsRoute.php

```diff
@@ -37,11 +37,24 @@ public function getDecorated(): AbstractCmsRoute
      * @Since("6.2.0.0")
      * @OA\Post(
      *      path="/cms/{id}",
-     *      summary="Resolves a cms page",
+     *      summary="Fetch and resolve a CMS page by criteria.",
+     *      description="Loads a content management page by its identifier and resolve the slot data. This could be media files, product listing and so on.",
```

Comment:
Mention, that the criteria passed is used to control what is loaded for the CMS page **and** also for the listing within the CMS page (if there is one).

Reply:
summary="Fetch and resolve a CMS page" (without the "." at the end)